NanoAnalysis/test/runLocal.py

```diff
@@ -100,13 +101,26 @@
     setConf("LEPTON_SETUP", 2022)
     setConf("IsMC", True)
     setConf("store","root://cms-xrd-global.cern.ch/")
+    #setConf("store","")
     setConf("APPLY_QCD_GGF_UNCERT", True) # for ggH
     setConf("fileNames",[
-        "/store/mc/Run3Summer22EENanoAODv12/GluGluHtoZZto4L_M-125_TuneCP5_13p6TeV_powheg2-JHUGenV752-pythia8/NANOAODSIM/130X_mcRun3_2022_realistic_postEE_v6-v2/2540000/25c8f5ff-9de0-4a0c-9e2f-757332ad392f.root",
+    #    "/eos/user/n/nipinto/nanoAODs/ggH125_2022_nano.root"
+      "/store/mc/Run3Summer22EENanoAODv12/GluGluHtoZZto4L_M-125_TuneCP5_13p6TeV_powheg2-JHUGenV752-pythia8/NANOAODSIM/130X_mcRun3_2022_realistic_postEE_v6-v2/2540000/25c8f5ff-9de0-4a0c-9e2f-757332ad392f.root",
 #        "/store/mc/Run3Summer22EENanoAODv12/GluGluHtoZZto4L_M-125_TuneCP5_13p6TeV_powheg2-JHUGenV752-pythia8/NANOAODSIM/130X_mcRun3_2022_realistic_postEE_v6-v2/2530000/8f306f2b-1284-41b8-a98f-744267f64b9c.root",
         ])
 #    json = {"1": [[1245, 1245],[1306, 1306],[1410, 1410],[1692, 1692],[1903, 1903],[1910, 1910],[1915, 1915],[1927, 1927],[1939, 1939],[1940, 1940],[1944, 1944],[1945, 1945],[1956, 1956],[1960, 1960],[1965, 1965],[1967, 1967],[1968, 1968],[1969, 1969],[2104, 2104]]}
-
+###################################################################################
+elif SampleToRun == "MELA_Test" : 
+    #ZH sample to test MELA categorization
+    setConf("SAMPLENAME", "ggH125")
+    setConf("LEPTON_SETUP", 2022)  # why 2023 doesn't work ? 
+    setConf("XSEC", 290.58626*0.0002745)
+    setConf("IsMC", True)
+    setConf("ADD_ALLEVENTS", True)
+    #setConf("NANOVERSION", 9)
```

Comment:
... and you would have to set this to 15 (although at some point we would need to add some other v15-specific tweaks that would not be compatible with your v12+LHEfix files, but I think that's not yet the case)